equinix/fabric_mapping_helper.go

```diff
@@ -1228,21 +1226,3 @@ func getCloudRouterUpdateRequest(conn v4.CloudRouter, d *schema.ResourceData) (v
 	}
 	return changeOps, nil
 }
-
```

Comment:
This code is needed in Config for some user-agent magic, so I moved it to `internal/fabric_utils`